style-bibliographique-des-collections-des-shp-ephe-paris.csl

```diff
@@ -466,8 +466,10 @@
       </else-if>
     </choose> 
   </macro>
-  <macro name="isbn">
+  <macro name="ISBN">
+   <group>  
     <text variable="ISBN" prefix=". ISBN : " suffix="."/>
+   </group> 
   </macro>
```

Comment:
Okay, if you want to show the ISBN in the list of references then you have to use a solution as you proposed. The automatic check is happy now, yeah! 

Soon someone will closer at your work here.